lib/runtime_tools/src/observer_backend.erl

```diff
@@ -547,7 +548,8 @@ etop_collect(Collector) ->
     %% utilization in etop). Next time the flag will be true and then
     %% there will be a measurement.
     SchedulerWallTime = erlang:statistics(scheduler_wall_time),
-    ProcInfo = etop_collect(processes(), []),
+    Iter = erlang:processes_iterator(),
```

Comment:
As you are using this function, you need to update the `runtime_dependencies` version of erts in `runtime_tools.app.src` to `"erts-@OTP-19369@"`.

Reply:
Fixed